### PR TITLE
Rename secretKey to privateKey

### LINKED ISSUE #15: Rename `secretKeyBase58` => `privateKeyBase58`

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Renamed `secretKey` to `privateKey` properties, to match Digital Bazaar conventions.
Closes #15.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # did-io ChangeLog
 
+## 0.8.0
+(Migrating from v0.7.0 will require renaming the relevant properties in existing
+DID Docs.)
+
+## Changed
+
+- (BREAKING) Updated terminology to match latest VeresOne DID method specs:
```
Comment (davidlehn) [inline CHANGELOG.md]: Minor nit, we've been using a bold style markup like:
- \*\*BREAKING**: ...

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+  `grantCapability -> capabilityDelegation`,
+  `invokeCapability -> capabilityInvocation`
+- (BREAKING) Renamed `privateKey` to `secretKey` properties, to match Digital
+  Bazaar conventions.
```
Comment (davidlehn) [inline CHANGELOG.md]: The PR title and this are written backwards.  It's also functions.  Perhaps:
```Renamed `secretKey` to `privateKey`, to match Digital Bazaar conventions.```

File: tests/veres-one/veres-one.spec.js [hunk 3]
```diff
@@ -172,12 +172,12 @@ describe('methods/veres-one', () => {
...
       const grantPublicKey = didDocument.doc.capabilityDelegation[0].publicKey[0];
       const creator = grantPublicKey.id;
-      const {secretKeyPem} = await didDocument.keys[grantPublicKey.id].export();
+      const {privateKeyPem} = await didDocument.keys[grantPublicKey.id].export();
 
       didDocument = await v1.attachDelegationProof({
         didDocument,
         creator,
-        privateKeyPem: secretKeyPem
+        privateKeyPem: privateKeyPem
```
Comment (mattcollier) [inline tests/veres-one/veres-one.spec.js]: just `privateKeyPem` in all the places.

**Review by mattcollier — APPROVED** [review]
Just a style nit.  TY!

File: CHANGELOG.md (hunk 2 above)
Comment (dmitrizagidulin) [reply CHANGELOG.md]: Whoops, thanks!

File: CHANGELOG.md (hunk 1 above)
Comment (dmitrizagidulin) [reply CHANGELOG.md]: Got it, thanks!

File: tests/veres-one/veres-one.spec.js (hunk 3 above)
Comment (dmitrizagidulin) [reply tests/veres-one/veres-one.spec.js]: Thanks :)

**Pull request merged**